.gitignore

```diff
@@ -22,4 +22,6 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-.eslintcache
\ No newline at end of file
+.eslintcache
+package-lock.json
```

Comment:
Why did you ignore the package-lock.json file? This is a really bad practice. We want to be able to control the versions of the library we use.